secgroup_interactive_egress.tf

```diff
@@ -0,0 +1,115 @@
+resource "openstack_networking_secgroup_v2" "interactive-egress" {
```

Comment:
This is 1:1 as we had it in the old cloud right?

Reply:
Yup, I created a 1:1.